code/game/objects/structures/curtains.dm

```diff
@@ -11,6 +11,11 @@
 	anchored = TRUE //curtains start secured in place
 	build_amt = 2
 	var/manipulating = FALSE //prevents queuing up multiple deconstructs and returning a bunch of cloth
+	var/curtain_material = MATERIAL_CLOTH
```

Comment:
Why do we need this instead of just using material?

Reply:
For some reason in initial testing just setting material didn't like. work. will look at

Reply:
This seems to be a common issue. A lot of code does this form of bypassing:
```dm
/obj/structure/railing/Initialize()
	. = ..()

	if(!isnull(material) && !istype(material))
		material = SSmaterials.get_material_by_name(material)
```

This should maybe be added as an Initialize override somewhere in the obj tree maybe? But otherwise, this works out.